### ISSUE TITLE
Transifex: "No PHP array pattern was found!"

### ISSUE THREAD

**Issue by devilx** [twostairs/paperwork#185]
I'm receiving e-mails from Transifex regarding the translations:

```
We have not been able to update the source file for the resource notebooks.php of the project Paperwork.
The URL we used to update the source file from was https://raw.githubusercontent.com/twostairs/paperwork/master/frontend/app/lang/en/notebooks.php.
The exact error we encountered was:
No PHP array pattern was found!
```

Maybe someone could help me here, as I don't really know Transifex?

**Comment by joshlemer** [comment]
I can't find anywhere that indicates they specifically don't allow concatinated strings (such as is found in the array's "welcome_note_content"), but I have a feeling this ha something to do with it. I don't know how to test it but I'd maybe try putting the entire content in one very long, un-concatinated string.

**Comment by devilx** [comment]
I've contacted support, they're looking into this issue.

**Comment by Liongold** [comment]
@devilx Did you hear something from them?

**Comment by devilx** [comment]
@Liongold yes, it actually looks like they fixed this, as I didn't receive any error mail again. But I'm not 100% sure yet, therefor this is still open.

**Comment by Baldrs** [comment]
Fixed that in #342

**Comment by Baldrs** [comment]
PS: Transifex parser does not understand PHP string concatenation
